packages/web-components/src/components/va-link/va-link.css

```diff
@@ -22,6 +22,10 @@
   font-weight: 700;
 }
 
+:host([icon-name])::part(icon) {
```

Comment:
I don't think that we need to use `::part` do we? That's for teams to pierce the shadow dom for styling. Can we just reference the icon element?

```css
va-icon { margin-right: 4px }
```